### PR TITLE
Convert "Viewing Structured Data".

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: src/_langs/en/tools/javascript/console/structured-data.markdown [hunk 1]
```diff
@@ -3,8 +3,8 @@ rss: false
 layout: article
 title: "View Structured Data in Console"
 seotitle: "View Structured Data in Chrome DevTools Console"
-description: "TBD description."
-introduction: "TBD introduction."
+description: "Use the `table()` method to compare data objects."
```
Comment (Meggin) [inline src/_langs/en/tools/javascript/console/structured-data.markdown]: Invert sentence to push action first. Also, can we put full method name here and directly refer to the Chrome DevTools Console.

File: src/_langs/en/tools/javascript/console/structured-data.markdown [hunk 2]
```diff
@@ -15,20 +15,54 @@ priority: 0
 collection: console
 key-takeaways:
   tldr-tbd:
-    - TBD tldr.
+    - Compare similar data objects using `table()`.
```
Comment (Meggin) [inline src/_langs/en/tools/javascript/console/structured-data.markdown]: We will want to link this to console API reference docs, and put in complete method name her.

File: src/_langs/en/tools/javascript/console/structured-data.markdown [hunk 3]
```diff
@@ -15,20 +15,54 @@ priority: 0
...
+    - Build an array to define property strings
```
Comment (Meggin) [inline src/_langs/en/tools/javascript/console/structured-data.markdown]: Missing full-stop.

File: src/_langs/en/tools/javascript/console/structured-data.markdown [hunk 4]
```diff
@@ -15,20 +15,54 @@ priority: 0
 collection: console
 key-takeaways:
   tldr-tbd:
```
Comment (Meggin) [inline src/_langs/en/tools/javascript/console/structured-data.markdown]: Change name here to be console instead of tldr-tbd.

File: src/_langs/en/tools/javascript/console/structured-data.markdown [hunk 5]
```diff
@@ -15,20 +15,54 @@ priority: 0
...
+    - Build an array to define property strings
 remember:
   note-tbd:
     - TBD note.
 ---
 {% wrap content %}
 
-TBD. Doc: https://developer.chrome.com/devtools/docs/console#viewing-structured-data
-
 {% include modules/takeaway.liquid list=page.key-takeaways.tldr-tbd %}
```
Comment (Meggin) [inline src/_langs/en/tools/javascript/console/structured-data.markdown]: Assuming name changes above, change here too to console instead of tldr-tbd.

File: src/_langs/en/tools/javascript/console/structured-data.markdown [hunk 6]
```diff
@@ -15,20 +15,54 @@ priority: 0
...
 
-### TBD
+### Viewing Structured Data
```
Comment (Meggin) [inline src/_langs/en/tools/javascript/console/structured-data.markdown]: Make this header two.

File: src/_langs/en/tools/javascript/console/structured-data.markdown [hunk 7]
```diff
@@ -15,20 +15,54 @@ priority: 0
...
+
+The `table()` method provides an easy way to view similar data objects. This will take the properties of an object and create a header. Row data comes from each indexes' properties value.
+
+#### A table in the console using two arrays.
```
Comment (Meggin) [inline src/_langs/en/tools/javascript/console/structured-data.markdown]: Header 3

File: src/_langs/en/tools/javascript/console/structured-data.markdown [hunk 8]
```diff
@@ -15,20 +15,54 @@ priority: 0
...
+    console.table([{a:1, b:2, c:3}, {a:"foo", b:false, c:undefined}]);
+    console.table([[1,2,3], [2,3,4]]);
+  
+The output of the example code:
+
+<img src="imgs/table-arrays.png" class="center" alt="console table display"/>
+
+The second parameter to table() is optional. You may define an array containing the property strings you wish to display.
+
+#### A console table using a collection of objects.
```
Comment (Meggin) [inline src/_langs/en/tools/javascript/console/structured-data.markdown]: Header 3

File: src/_langs/en/tools/javascript/console/structured-data.markdown [hunk 9]
```diff
@@ -15,20 +15,54 @@ priority: 0
...
+    family.father = new Person("John", "Doyle", 33);
+    family.daughter = new Person("Lily", "Doyle", 5);
+    family.son = new Person("Mike", "Doyle", 8);
+  
+    console.table(family, ["firstName", "lastName", "age"]);
+
+The output of the example code:
 
-TBD.
+<img src="imgs/table-people-objects.png" class="center" alt="console output with table objects"/>
```
Comment (Meggin) [inline src/_langs/en/tools/javascript/console/structured-data.markdown]: This needs to change to markdown format . And should map to the image that you've added, which I think is images folder?

File: src/_langs/en/tools/javascript/console/structured-data.markdown [hunk 10]
```diff
@@ -15,20 +15,54 @@ priority: 0
...
 
 {% include modules/remember.liquid title="Remember" list=page.remember.note-tbd %}
```
Comment (Meggin) [inline src/_langs/en/tools/javascript/console/structured-data.markdown]: This was a placeholder for a note, but you don't have any, so this can be removed.

File: src/_langs/en/tools/javascript/console/structured-data.markdown (hunk 2 above)
Comment (jpmedley) [inline src/_langs/en/tools/javascript/console/structured-data.markdown]: I just verified with grep that "console\.table" is not documented.

File: src/_langs/en/tools/javascript/console/structured-data.markdown (hunk 9 above)
Comment (jpmedley) [inline src/_langs/en/tools/javascript/console/structured-data.markdown]: Will do, but you should know I was copying what what was done in another file.
